fastlane/Fastfile

```diff
@@ -178,43 +178,66 @@ platform :ios do
     end
     UI.user_error!("Failed to determine target repository name") if target_repository_name.nil? || target_repository_name.empty?
 
-    # Loop over all PNG files in the tmp_dir, parse the offering ID from the file names, and move them into the target repository
-    output_dir = record_paywall_screenshots
+    # Loop over all platforms and generate screenshots for each
+    platforms = ["ios", 
+                # These platforms work locally but we have been unable to figure out how to get them to work on CI due to codesigning issues.
+                #  "iphone-app-on-mac",
+                #  "ipad-app-on-mac", 
```

Comment:
Are these supposed to be commented out? I ask because the PR title says that it also adds iPad/iPhone app on Mac validation